ui/apps/platform/src/Containers/MainPage/MainPage.tsx

```diff
@@ -63,22 +62,22 @@ function MainPage(): ReactElement {
     const hasServiceIdentityWritePermission = hasReadWriteAccess('ServiceIdentity');
 
     return (
-        <AppWrapper>
+        <>
+            <Notifications />
             <PublicConfigHeader />
-            <div className="flex flex-1 flex-col h-full relative">
-                <AnnouncementBanner />
-                <ServerStatusBanner />
-                <Notifications />
-                <CredentialExpiryBanner
-                    component="CENTRAL"
-                    hasServiceIdentityWritePermission={hasServiceIdentityWritePermission}
-                />
-                <CredentialExpiryBanner
-                    component="SCANNER"
-                    hasServiceIdentityWritePermission={hasServiceIdentityWritePermission}
-                />
-                <OutdatedVersionBanner />
-                <DatabaseStatusBanner />
+            <AnnouncementBanner />
+            <CredentialExpiryBanner
+                component="CENTRAL"
+                hasServiceIdentityWritePermission={hasServiceIdentityWritePermission}
+            />
+            <CredentialExpiryBanner
+                component="SCANNER"
+                hasServiceIdentityWritePermission={hasServiceIdentityWritePermission}
+            />
+            <OutdatedVersionBanner />
+            <DatabaseStatusBanner />
+            <ServerStatusBanner />
```

Comment:
Just out of curiosity, why did you move the ServerStatusBanner to the end of the list?

Reply:
Good eyes.

Notice in the baseline pair of pictures how the yellow transient status banner gets lost among the certificate expiry banners which might remain for hours or days?

The banners are in order of increasing timeliness of the information.